OpenRA.Mods.Common/Warheads/DamageWarhead.cs

```diff
@@ -21,7 +21,7 @@ public abstract class DamageWarhead : Warhead
 		[Desc("How much (raw) damage to deal.")]
 		public readonly int Damage = 0;
 
-		[Desc("Types of damage that this warhead causes. Leave empty for no damage.")]
+		[Desc("ProductionTypes of damage that this warhead causes. Leave empty for no damage.")]
```

Comment:
And another.

Reply:
Well I guess I have my work cut out for me.